### PR TITLE
recover stability of arm IK

### PR DESCRIPTION
**Pull request by AndrewMeadows**
* remove twist constraints on hands
* tighten swing constraint on shoulder (this reintroduces a bug where IK can clamp the input animation, but we'll fix that again later a different way)

### CODE REVIEW

File: examples/tests/particleOrientationTest.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ function emitter(jointName) {
             x:0,
             y: 0,
             z: 0,
-            w: Math.PI
+            w: 1
```
Comment (AndrewMeadows) [inline examples/tests/particleOrientationTest.js]: The identity quaternion should be normalized.  This was a bug in an example JS script I noticed when doing something else.  I decided to piggyback this fix into an unrelated PR.

File: libraries/animation/src/AnimInverseKinematics.cpp [hunk 2]
```diff
@@ -627,6 +627,9 @@ void AnimInverseKinematics::initConstraints() {
         } else if (0 == baseName.compare("Hand", Qt::CaseSensitive)) {
             SwingTwistConstraint* stConstraint = new SwingTwistConstraint();
             stConstraint->setReferenceRotation(_defaultRelativePoses[i].rot);
+            stConstraint->setTwistLimits(0.0f, 0.0f); // max == min, disables twist limits
```
Comment (AndrewMeadows) [inline libraries/animation/src/AnimInverseKinematics.cpp]: The **SwingTwistConstraint** API has been changed so that if the max/min twist limits are identical then there is no enforcement of twist, and it behaves like a pure **SwingConstraint**.

File: libraries/animation/src/AnimInverseKinematics.cpp [hunk 3]
```diff
@@ -670,7 +674,7 @@ void AnimInverseKinematics::initConstraints() {
             stConstraint->setTwistLimits(-MAX_SHOULDER_TWIST, MAX_SHOULDER_TWIST);
 
             std::vector<float> minDots;
-            const float MAX_SHOULDER_SWING = PI / 6.0f;
+            const float MAX_SHOULDER_SWING = PI / 20.0f;
```
Comment (AndrewMeadows) [inline libraries/animation/src/AnimInverseKinematics.cpp]: I'm restoring the old (tight) swing limit on the shoulder for more stable IK.  This reintroduces a problem where the IK system can "clamp" the input base animation (aka the "underpose") when it tries to violate the constraints.  I've got an idea on how to fix this problem, but will do that in a separate PR.

**Comment by AndrewMeadows** [comment]
https://app.asana.com/0/26225263936266/97505974353476

To test: 

(1) Login with Vive or use hydra
(2) Put the hand controllers in strange and difficult positions so that avatar contorts drastically
--> this PR should be more stable than the previous version

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9bf4a30a232b9008c741effe53263789101bd3f5 available at: https://gist.github.com/3a50fd1ee85f410acaab<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7246.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7246.dmg

**Comment by AndrewMeadows** [comment]
test this please

**Comment by PhilipRosedale** [comment]
Reviewed and this version is more stable for me on Hydras.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9bf4a30a232b9008c741effe53263789101bd3f5 available at: https://gist.github.com/3a50fd1ee85f410acaab<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7246.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7246.dmg
